pandas/tests/frame/test_alter_axes.py

```diff
@@ -383,7 +383,7 @@ class Thing(frozenset):
             def __repr__(self) -> str:
                 tmp = sorted(self)
                 # double curly brace prints one brace in format string
-                return "frozenset({{{}}})".format(", ".join(map(repr, tmp)))
+                return f"frozenset({{{', '.join(map(repr, tmp))}}})"
```

Comment:
Can you take the "join" outside the f-string?

Reply:
Can you please elaborate more? I did not understand your idea

Reply:
`joined_reprs = ', '.join(map(repr, tmp))` should be fine